data/strings/types_strings.txt

```diff
@@ -29859,21 +29859,49 @@
     zh-Hans = 景点
     zh-Hant = 景點
 
-  [type.tourism.attraction.animal]
+  [type.attraction.animal]
     ref = type.tourism.attraction
```

Comment:
ah, please remove this `ref`, I haven't noticed it before :/

it basically means "take translation from tourism.attraction if they're not specified here explicitly"

that should fix wrong translations for pt-BR and es-MX

Reply:
Thanks! That solved the wrong pt-BR, es-MX and en-GB translations